.circleci/config.yml

```diff
@@ -470,8 +470,8 @@ workflows:
             - dev-build
           filters:
             branches:
-              only:
-                - /^bug\/flaky-test-.*$/ # only run go tests on bug/flaky-test-* for now since we are fixing tests
+              ignore:
+                - /^pull\/.*$/ # only run link checker on non forks
```

Comment:
I don't follow the comment about the lint checker here, isn't this filter for `go test`?

Reply:
copy-pasted without changing the comment 🤦‍♂